app/addon_cmds/commands.py

```diff
@@ -159,6 +168,12 @@ def addon(
         )
 
     ctx.obj["addons_dict"] = addons_dict
+    if any(addon_name not in addons_dict.keys() for addon_name in rosa):
+        click.echo(
+            "An addon indicated with --rosa does not match any of addons names that were given.\nPlease fix "
+            "typo."
```

Comment:
Please remove `Please fix typo.` from the error
Please log the passed addons as well

Reply:
Done